### PR TITLE
DOCS-7286: CSM Fargate

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do? What is the motivation?

Private beta for CSM Fargate.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/cloud_security_management/setup/fargate.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+## Prerequisites
+
+* [CSM Enterprise][4] or [CSM Workload Security][5] with the AWS integration configured
+* Access to AWS Management Console
+* AWS Fargate ECS or EKS workloads
+
+### Images
+
+* cws-instrumentation: datadog/cws-instrumentation-dev:cws-instrumentation-beta
+* Datadog-agent: datadog/agent:7.51.0-rc.6-linux
```
Comment (safchain) [inline content/en/security/cloud_security_management/setup/fargate.md:24]: now that the 7.51 is out, we can put `datadog/agent:latest` here

**Comment by safchain** [comment]
We should remove the mention to "network", "captures traffic metadata" in the schema as we don't support them yet.

File: content/en/security/cloud_security_management/setup/fargate.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+* cws-instrumentation: datadog/cws-instrumentation-dev:cws-instrumentation-beta
+* Datadog-agent: datadog/agent:latest
+
+## Installation
+
+### AWS Console
+
+1. Log in to the [AWS Management Console][6].
+2. Navigate to the ECS section.
+3. On the left menu, click **Task Definitions**, then click **Create new Task Definition with JSON** or choose an existing Fargate task definition.
```
Comment (aliciascott) [inline content/en/security/cloud_security_management/setup/fargate.md:32]:
```suggestion
3. On the left menu, click **Task Definitions**, then click **Create new Task Definition with JSON**, or choose an existing Fargate task definition.
```

File: content/en/security/cloud_security_management/setup/fargate.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+4. Use the JSON definition of the AWS CLI method.
```
Comment (aliciascott) [inline content/en/security/cloud_security_management/setup/fargate.md:33]:
```suggestion
4. Use the JSON definition, or the AWS CLI method.
```
Is this supposed to say "or" the AWS CLI method ? Also would then suggest an anchor link to the AWS CLI method below.

**Review by aliciascott — APPROVED** [review]
Looks good! Just a couple small suggestions but approved :)

File: content/en/security/cloud_security_management/setup/fargate.md (hunk 3 above)
Comment (drichards-87) [reply content/en/security/cloud_security_management/setup/fargate.md:33]: Good catch. After checking AWS Console, I'm pretty sure it's the latter.

**Pull request merged**